configs/neox_arguments.md

```diff
@@ -1680,6 +1680,14 @@ Args for deepspeed config
 
 
 
+- **bf16**: dict
```

Comment:
We already support the `precision` config option. Instead of making a new config for bf16, let's make `bf16` an allowable value for `precision` to take on.